cmd/cerbosctl/internal/flags.go

```diff
@@ -180,6 +182,8 @@ func (lpfd *ListPoliciesFilterDef) FlagSet() *pflag.FlagSet {
 	fs := pflag.NewFlagSet("filters", pflag.ExitOnError)
 	fs.StringArrayVar(&lpfd.fieldEq, "field-eq", []string{}, "Filter a field with an exact match")
 	fs.StringArrayVar(&lpfd.fieldMatch, "field-match", []string{}, "Filter a field with a regex match")
+	fs.StringVar(&lpfd.sortAsc, "sort-asc", "", "Sort the output by ascending order (available sorting targets: name, version)")
+	fs.StringVar(&lpfd.sortDesc, "sort-desc", "", "Sort the output by descending order (available sorting targets: name, version)")
```

Comment:
Because there's no support for multi-level sorting (e.g. by name then by version), I think it would be clearer to use `--sort` to get the column to sort by and an optional boolean flag `--sort-desc` to reverse the order.